Dockerfile

```diff
@@ -8,11 +8,8 @@ COPY prepare-image.sh /bin/
 RUN prepare-image.sh && \
   rm -f /bin/prepare-image.sh
 
-COPY ./inspector.conf.j2 /etc/ironic-inspector/ironic-inspector.conf.j2
-COPY ./runironic-inspector.sh /bin/runironic-inspector
-COPY ./runhealthcheck.sh /bin/runhealthcheck
-COPY ./ironic-common.sh /bin/ironic-common.sh
-COPY ./runlogwatch.sh /bin/runlogwatch.sh
+COPY inspector.conf.j2 /etc/ironic-inspector/
```

Comment:
Renaming `ironic-inspector.conf.j2` to `inspector.conf.j2` without changing [the script that refers to it](https://github.com/metal3-io/ironic-inspector-image/blob/master/runironic-inspector.sh#L3) will break.

Reply:
thanks, fixed